.github/workflows/aws.yml

```diff
@@ -0,0 +1,36 @@
+name: Start/Stop AWS instance
```

Comment:
I'm trying to introduce this file in https://github.com/intel/llvm/pull/9875 as well. Hopefully #9875 could be merged first and it would disappear from this PR.